pkgs/development/python-modules/denonavr/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchFromGitHub, requests, tox }:
+
+buildPythonPackage rec {
+  pname = "denonavr";
+  version = "0.7.10";
+
+  src = fetchFromGitHub {
+    owner = "scarface-4711";
+    repo = "denonavr";
+    rev = version;
+    sha256 = "078nhr69f68nfazhmkf2sl7wiadqx96a5ry3ziggiy1xs04vflj7";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  doCheck = true;
+  checkInputs = [ tox ];
```

Comment:
we don't need tox in nix, nix does that for us (abstracts over python version), instead:
```
  checkInputs = [ pytest ];
  checkPhase = ''
    pytest tests
  '';
```